dora/core/server/worker/src/main/java/alluxio/worker/http/HttpRequestUtil.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.worker.http;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Util class for handling HTTP request.
+ */
+public final class HttpRequestUtil {
+
+  /**
+   * Parse the parameters from the HTTP request URI and return a map of the parameters.
+   *
+   * @param requestUri the HTTP request URI
+   * @return a map of the parameters
+   */
+  public static Map<String, String> parseRequestParameters(String requestUri) {
+    requestUri = requestUri.substring(requestUri.indexOf("?") + 1);
+    String[] params = requestUri.split("&");
+    Map<String, String> parametersMap = new HashMap<>();
+    for (String param : params) {
+      String[] keyValue = param.split("=");
+      parametersMap.put(keyValue[0], keyValue[1]);
+    }
+    return parametersMap;
+  }
+
+  /**
+   * Parse the HTTP request URI and extract the fields.
+   *
+   * @param requestUri the HTTP request URI
+   * @return a list of fields
+   */
+  public static List<String> extractFieldsFromHttpRequestUri(String requestUri) {
+    int paramIndex = requestUri.indexOf("?");
```

Comment:
Just a quick question: Could the fileId contain a question mark(?)?

Reply:
File ID cannot contain a question mark(?). This is just for parsing a URI with parameters. For example, we may need to parse this URI `http://127.0.0.1:28080/v1/file/5f2829f08879b0e89d07174cffa8d891bdf08ba9e91218e30fe39503dd42e32c/page/0?offset=5&length=7` in the future.

Reply:
I think file id should be a md5 or sha256 hashed value, am I right? @JiamingMai so there should be no question mark.

Reply:
But the dir path might contains a question mark, I think we should escape or encode the question mark in the path